### PR TITLE
Include bdist_rpm

### PR DESCRIPTION
**Pull request by alvarosimon**

### CODE REVIEW

File: bdist_rpm.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+VERSION=$(grep "Version:.*[0-9]" singularity.spec | tr -s " " |  awk '{print $2;}')
+GITTAG=$(git log --format=%ct.%h -1)
+
+# Copy required files
+cp "singularity-${VERSION}.tar.gz" "SOURCES"
+cp singularity.spec "SPECS"
+
+
+rpmbuild --define "gittag ${GITTAG}" --define "_topdir $PWD" -ba SPECS/singularity.spec
```
Comment (stdweird) [inline bdist_rpm.sh]: gittag is only picked up when actually used in spec

Comment (alvarosimon) [reply bdist_rpm.sh]: true, I have included the missing `%{gittag}`   into the `singularity.spec.in`    layout

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Ah, this went to the wrong branch @alvarosimon, you should redo this for the 2.4.5 branch (I just made that the default).

**Pull request merged**